docs/testing/debug-with-vs-extension.md

```diff
@@ -18,67 +24,51 @@ This debugging mode is dynamic, allowing you to set breakpoints while code is ru
 
-1. If you need to create an add-in project, [use the Yo Office generator to create one](../quickstarts/excel-quickstart-jquery.md?tabs=yeomangenerator). Follow the prompts within the command line to set up your project. You can choose any language or type of project to suit your needs. This tutorial uses an Excel task pane add-in.
-
-    > [!NOTE]
-    > If you already have a project, skip step 1 and move to step 2.
-
-1. Open a command prompt as administrator.
-   ![Command prompt options, including "run as administrator" in Windows 10 and 11.](../images/run-as-administrator-vs-code.jpg)
-
-1. Navigate to your project directory.
-
-1. Run the following command to open your project in Visual Studio Code as an administrator.
-
-    ```command&nbsp;line
-    code .
-    ```
+These instructions assume you have experience using the command line, understand basic JavaScript, and have created an Office Add-in project before using the Yo Office generator. If you haven't done this before, consider visiting one of our tutorials, like this [Excel Office Add-in tutorial](../tutorials/excel-tutorial.md).
 
-  Once Visual Studio Code is open, navigate manually to the project folder.
+1. If you need to create an add-in project to experiment with debugging in Visual Studio Code, [use the Yo Office generator to create one](../quickstarts/excel-quickstart-jquery.md?tabs=yeomangenerator). Follow the prompts within the command line to set up your project. You can choose any language or type of project to suit your needs. If you want to debug an existing project, skip to the next step.
 
-  > [!TIP]
-  > To open Visual Studio Code as an administrator, select the **run as administrator** option when opening Visual Studio Code after searching for it in Windows.
+1. Open VS Code *as an administrator* and open your project in it. 
 
 1. Within VS Code, select **Ctrl+Shift+X** to open the Extensions bar. Search for the "Microsoft Office Add-in Debugger" extension and install it.
 
-1. In the .vscode folder of your project, open the **launch.json** file. Add the following code to the `configurations` section.
+1. Next, choose  **View > Run** or enter **Ctrl+Shift+D** to switch to debug view.
 
-    ```JSON
-    {
-      "type": "office-addin",
-      "request": "attach",
-      "name": "Attach to Office Add-ins",
-      "port": 9222,
-      "trace": "verbose",
-      "url": "https://localhost:3000/taskpane.html?_host_Info=HOST$Win32$16.01$en-US$$$$0",
-      "webRoot": "${workspaceFolder}",
-      "timeout": 45000
-    }
-    ```
+1. From the **RUN AND DEBUG** options, choose the Edge Legacy option for your host application, such as **Excel Desktop (Edge Legacy)**. Select **F5** or choose **Run > Start Debugging** from the menu to begin debugging. This action automatically launches a local server in a Node window to host your add-in and then automatically opens the host application, such as Excel or Word. This may take several seconds.
 
```

Comment:
I noticed a comment about project not built with the yeoman generator earlier . . . those projects wouldn't have the launch.json configs mentioned here and they would have to add them.

Reply:
Done

Reply:
Fixed. Added appendix to explain this.